### DISCUSSION TITLE
ccf-validation-tools/Peripheral_nervous_system/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#326] [General]
# ccf-validation-tools/Peripheral_nervous_system/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Peripheral_nervous_system/comments/

**Comment by emquardokus** [comment]
Peripheral_Nervous_System_v1.2 https://docs.google.com/spreadsheets/d/1ufuw7PGcOVU1sxfIB4E_jesk8Hk0zJVGPPlNeuU0gzY/edit#gid=917578386

**Different labels**

1) There does NOT appear to be any trailing space  in the ASCT+B table and if there is a preceding space in the Ubergraph, shouldn't that also be fixed? The term UBERON:0001813 has a different name/label in the source ontology in the following 1 row 76. The name/label in the ASCT+B table is spinal nerve plexus _ and the one in the ontology is _spinal nerve plexus. For reference, the given name/label by SMEs is spinal nerve plexus . Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

2) FIXED to match ontology:   The term UBERON:0005360 has a different name/label in the source ontology in the following 1 row 1239. The name/label in the ASCT+B table is inferior glossopharyngeal IX ganglion (petrosal ganglion) and the one in the ontology is inferior glossopharyngeal IX ganglion. For reference, the given name/label by SMEs is inferior glossopharyngeal IX ganglion (petrosal ganglion). Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Peripheral_Nervous_System_v1.2 https://docs.google.com/spreadsheets/d/1ufuw7PGcOVU1sxfIB4E_jesk8Hk0zJVGPPlNeuU0gzY/edit#gid=917578386

**Different labels**

There were "returns" after plexus; those have been removed for all rows 77 to 81. The term UBERON:0001813 has a different name/label in the source ontology in the following 1 row 76. The name/label in the ASCT+B table is spinal nerve plexus _ and the one in the ontology is _spinal nerve plexus. For reference, the given name/label by SMEs is spinal nerve plexus . Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
